webpack.config.js

```diff
@@ -1,11 +1,12 @@
 // Hint: if something doesn't work as expected: yarn upgrade --latest
 
 var Encore = require('@symfony/webpack-encore');
+var WorkboxPlugin = require('workbox-webpack-plugin');
 var webpack = require('webpack');
 
 Encore
     .setOutputPath('public/build/')
-    .setPublicPath('build/')
+    .setPublicPath('/build')
```

Comment:
Besides all other changes, which I will have to look at when there is more time - this one is wrong.
The setting is used like this to allow usage in subdirectory scenarios.
The webpack/encore warning is highly annoying and misleading in this case.

Reply:
ah, wondered why this was changed. Just a draft PR as an idea how to get there :)